libr/util/ctype.c

```diff
@@ -482,7 +482,7 @@ R_API char *r_type_func_args_name(Sdb *TDB, const char *func_name, int i) {
 
 #define MIN_MATCH_LEN 4
 
-static char *type_func_try_guess(Sdb *TDB, char *name) {
+static R_OWN char *type_func_try_guess(Sdb *TDB, R_NONNULL char *name) {
```

Comment:
are we going to use these macros in `static` functions too? I'd be to use them only in R_API functions, static functions can change often.

Reply:
yes

Reply:
kk